File: app/src/main/java/com/yinshan/happycash/view/me/view/impl/AboutActivity.java
package com.yinshan.happycash.view.me.view.impl;

import android.support.v4.content.ContextCompat;
import android.widget.TextView;

import com.yinshan.happycash.R;
import com.yinshan.happycash.framework.BaseSingleActivity;

import butterknife.BindView;

/**
 * Created by huxin on 2018/3/20.
 */

public class AboutActivity extends BaseSingleActivity{

    @Override
    protected String bindTitle() {
        return getResources().getString(R.string.about_happy_cash);
    }

    @Override
    protected int bindDownLayout() {
        return R.layout.activity_about;
    }

    @Override
    protected void secondInit() {
        lowestBg.setBackgroundColor(ContextCompat.getColor(this,R.color.app_yellow));
    }
}

File: app/src/main/java/com/yinshan/happycash/view/me/view/impl/AboutActivity.kt
package com.yinshan.happycash.view.me.view.impl

import android.support.v4.content.ContextCompat

import com.yinshan.happycash.R
import com.yinshan.happycash.framework.BaseSingleActivity
import kotlinx.android.synthetic.main.activity_single_base.*

/**
 * Created by huxin on 2018/3/20.
 */

class AboutActivity : BaseSingleActivity() {

    override fun bindTitle(): String {
        return resources.getString(R.string.about_happy_cash)
    }

    override fun bindDownLayout(): Int {
        return R.layout.activity_about
    }

    override fun secondInit() {
        lowest_bg.setBackgroundColor(ContextCompat.getColor(this, R.color.app_yellow))
    }
}
